pkg/devfile/parser/parse_test.go

```diff
@@ -2983,6 +2983,7 @@ func Test_parseFromRegistry(t *testing.T) {
 			wantDevFile:   parentDevfile,
 			registryUrl:   registry,
 			registryId:    registryId,
+			wantErr:       true,
```

Comment:
test title should be `should NOT be able to parse from provided registryUrl without prefix`

Reply:
updated